src/main/java/com/questhelper/helpers/quests/zogreflesheaters/ZogreFleshEaters.java

```diff
@@ -271,15 +271,17 @@ public void setupSteps()
 		talkToZavistic = new NpcStep(this, NpcID.ZAVISTIC_RARVE, new WorldPoint(2598, 3087, 0), "Talk to Zavistic Rarve at the Yanille Wizards' Guild. If you don't have 66 Magic, ring the bell outside the guild.");
 		talkToZavistic.addDialogStep("I'm here about the sicks...err Zogres");
 
-		goUpToSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0), "Go upstairs to Sithik upstairs in north Yanille.");
+		goUpToSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0),
+			"Go upstairs to Sithik in a house north or the Wizards' Guild.");
```

Comment:
Bit of a restructuring of the sentence as a whole - I think this reads better (+ fixes the `or` typo)
```suggestion
			"Go upstairs and speak to Sithik in the house north of the Wizards' Guild.");
```